.eslintrc.js

```diff
@@ -78,11 +78,13 @@ module.exports = {
         },
       },
       rules: {
-        'require-atomic-updates': [0],
-        'import/no-named-as-default': 0,
+        'no-duplicate-imports': [0], // handled by @typescript-eslint
+        '@typescript-eslint/ban-types': [0], // TODO enable in future
```

Comment:
This rule forbids using empty object as type. Probably good idea to enable it in future.